simulation_parameters/Constants.cs

```diff
@@ -793,12 +793,12 @@ public static class Constants
     /// <summary>
     ///   The speed of which a cell can absorb compounds from digestible engulfed objects.
     /// </summary>
-    public const float ENGULF_COMPOUND_ABSORBING_PER_SECOND = 0.5f;
+    public const float ENGULF_COMPOUND_ABSORBING_PER_SECOND = 0.3f;
 
     /// <summary>
     ///   How much compounds a cell can absorb per second from digestible engulfed objects.
```

Comment:
Looks like this comment is copy pasted from above. Maybe you could fix this comment in this PR?

Reply:
It's not really copy-pasted in my eyes

Reply:
Okay, I misworded my comment. I should have said that the comment here didn't seem to make sense in relation to what the constant is. So the comment talks about "compound a cell can absorb per second" but the constant name contains "COMPOUND_ABSORPTION_YIELD". Yield is not the same as speed. So either the comment or the variable name has to change because they don't match.